go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/gjvnq/xpath
+
+go 1.12
+
+require github.com/antchfx/xpath v0.0.0-20190319080838-ce1d48779e67
```

Comment:
Wait, why do you add go mod in this patch that has nothing to do with moving the library to go mod? Have we discussed about the module management strategy moving forward and agreed upon on this approach?